simple_history/tests/tests/test_models.py

```diff
@@ -1851,6 +1852,11 @@ def test_separation(self):
         self.assertEqual(add.restaurants.all().count(), 0)
         self.assertEqual(add.places.all().count(), 0)
 
+    def test_self_field(self):
+        poll1 = PollWithSelfManyToMany.objects.create()
+        poll2 = PollWithSelfManyToMany.objects.create()
+        poll1.relations.add(poll2)
+
```

Comment:
It would be great if you could add some assertions here, e.g. on which history records were created, and maybe check that `poll2` was actually added to `poll1.relations` 🙂